pkg/config/loader_test.go

```diff
@@ -71,6 +71,9 @@ func TestDefaultResources(t *testing.T) {
 	if len(cfg.Resources) != 0 {
 		t.Error("Default resources should not be applied if specified in config")
 	}
+	if cfg.Resources == nil {
+		t.Error("Empty resources should not be converted to nil")
+	}
```

Comment:
This is the new test that ensures `sonobuoy master` will really load empty, non-nil resources.